tests/test_utils.py

```diff
@@ -777,6 +833,56 @@ def clean_athena_db(connection_string: str, db_name: str, table_to_keep: str) ->
         engine.dispose()
 
 
+def get_awsathena_db_name(db_name_env_var: str = "ATHENA_DB_NAME") -> str:
+    """Get awsathena database name from environment variables.
+
+    Returns:
+        String of the awsathena database name.
+    """
+    athena_db_name: str = os.getenv(db_name_env_var)
+    if not athena_db_name:
+        raise ValueError(
+            f"Environment Variable {db_name_env_var} is required to run integration tests against AWS Athena"
+        )
+    return athena_db_name
+
+
+def get_awsathena_connection_url(db_name_env_var: str = "ATHENA_DB_NAME") -> str:
+    """Get awsathena connection url from environment variables.
+
+    Returns:
+        String of the awsathena connection url.
+    """
+    ATHENA_DB_NAME: str = get_awsathena_db_name(db_name_env_var)
+    ATHENA_STAGING_S3: Optional[str] = os.getenv("ATHENA_STAGING_S3")
+    if not ATHENA_STAGING_S3:
+        raise ValueError(
+            "Environment Variable ATHENA_STAGING_S3 is required to run integration tests against AWS Athena"
+        )
+
+    return f"awsathena+rest://@athena.us-east-1.amazonaws.com/{ATHENA_DB_NAME}?s3_staging_dir={ATHENA_STAGING_S3}"
+
+
+def get_connection_string_and_dialect(
+    athena_db_name_env_var: str = "ATHENA_DB_NAME",
+) -> Tuple[str, str]:
+
+    with open("./connection_string.yml") as f:
+        db_config: dict = yaml_handler.load(f)
```

Comment:
Should this be `file_relative_path`? Will we ever run this from another location?

Reply:
Hmm good question. This method is usually invoked via a script that is typically run as a test using test_script_runner.py so this file is relative to the environment where this is invoked. I think there is definitely an opportunity to improve this integration test part of the codebase, maybe using containers instead of running each as a module. Would be worth discussing.